public/js/main.js

```diff
@@ -730,6 +743,11 @@ async function main() {
         handleError(e);
     }
 
+    var list = document.getElementsByClassName('tabulator-cell'); //support right to left text
```

Comment:
I think it's possible to directly apply this to the `<body>` tag instead of doing this for all cells

Reply:
I guess for the previous fix in Linkurious Enterprise they added is to every span. I tried for body it wasn't effective. I totally understand it's annoying to have it on every cell and having such piece of code 😞...

Reply:
Just realised I didn't check if it works with pagination though...

Reply:
Fixed